functions/src/utils/functions.js

```diff
@@ -51,6 +51,35 @@ const updateVideo = async (videoId, filePath, { description, title }) => {
         console.error(error);
     }
 };
+const getLocationdata = async location => {
+    const amount = await db.collection('locations').doc(location);
+    const documentData = await amount.get();
+    const data = documentData.data().price;
+    return Number(data);
+};
+const getLocationsAmount = async locationarray => {
+    const price = [];
+    locationarray.forEach(location => {
+        price.push(getLocationdata(location));
+    });
+    return await Promise.all(price);
+};
+const uploadMultipleImages = async (images, token) => {
+    const promises = [];
+    images.forEach(image => {
+        promises.push(uploadRequest(image, token));
+
+    });
+    return await Promise.all(promises);
+};
+const getMultipleFirebaseLink = async (images, token) => {
+    const promises = [];
+    images.forEach(image => {
+        promises.push(getFirebaseLink(image.originalname, token));
+    });
```

Comment:
Use this pattern for the above util functions as well
```
const promises = images.forEach(image => getFirebaseLink(image.originalname, token));
```

Reply:
ok i will now